app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [Required] Good job using `Rails Active Callbacks`, I think it would be much better if you made the `update_likes_counter` method private since it should only be called within this class. You can do this by refactoring your code as shown below.

```Ruby
class Like < ApplicationRecord
  belongs_to :author, class_name: 'User'
  belongs_to :post

  after_save :update_likes_counter
  
  private  

  def update_likes_counter
    post.increment!(:likes_counter)
  end
end

```

Reply:
Grateful for the review! @Felix45 , All changes implemented.